ivy/functional/frontends/tensorflow/signal.py

```diff
@@ -34,3 +36,10 @@ def vorbis_window(window_length, dtype=ivy.float32, name=None):
 def idct(input, type=2, n=None, axis=-1, norm=None, name=None):
     inverse_type = {1: 1, 2: 3, 3: 2, 4: 4}[type]
     return ivy.dct(input, type=inverse_type, n=n, axis=axis, norm=norm)
+
+@to_ivy_arrays_and_back
+def inverse_stft_window_fn(frame_step, forward_window_fn=tf.signal.hann_window,name=None):
+    return lambda frame_length: tf.signal.inverse_stft_window_fn(frame_length, forward_window_fn, frame_step)
```

Comment:
We don't use any framework in the frontends only Ivy functions are used to avoid dependency